heron/scheduler-core/src/java/com/twitter/heron/scheduler/SubmitterMain.java

```diff
@@ -374,13 +389,12 @@ public boolean submitTopology() {
 
       // create an instance of uploader
       uploader = ReflectionUtils.newInstance(uploaderClass);
+
     } catch (IllegalAccessException | InstantiationException | ClassNotFoundException e) {
-      LOG.log(Level.SEVERE, "Failed to instantiate instances", e);
-      return false;
+      throw new TopologySubmissionException(
+          String.format("Failed to instantiate instances: %s", e.getMessage()), e);
```

Comment:
used three `try...catch` clause: https://github.com/twitter/heron/pull/1571/commits/12a41b1836f0b799a30387f7f8602d2e35ca9a30 

mark this one as resolved unless there is more question